### ISSUE TITLE
Function to generate participant list from data folder

### ISSUE THREAD

**Issue by thartbm** [thartbm/PyVMEC#86]
Loop through all folders in the experiments data folder, and mark:
- (0) full participant: JSON matches experiment's JSON, data is complete
- (1) incomplete participant: there is a valid JSON, but data collection has not been completed
- (2) non-matching participants: there is a JSON and data (complete or incomplete), but the JSON doesn't match the experiment's JSON
- (3) invalid folders: there's no JSON and/or no data

Basically, the number indicates the level of wrongness. I guess this would result in a dictionary where each entry has a list of participant names:
{0: ['marius', 'julius'], 1: ['tryout1'], 2: ['old', 'version'], 3: ['icon_backup']}

This can then be used in other functions.

**Comment by juliusjgm12** [comment]
I made a function in GUI.py that does just this called generate_participant_markers. It creates the dictionary you detailed as self.participant_markers.
